rules/completed-docs/index.html

```diff
@@ -1,9 +1,9 @@
 ---
 ruleName: completed-docs
-description: Enforces documentation for important items be filled out.
+description: Enforces documentation and JSDoc comments for important items be filled out.
```

Comment:
@304NotModified Sorry for the delay

> but it also works for non-JS docs? (double slash)

After looking at the code and the tests for this rule, I'm pretty sure it only checks JSDoc.

That said, I'd prefer `Enforces JSDoc comments for important items be filled out.`
And as stated above, the changes should be done in the rule metadata in the source file instead.